### DISCUSSION TITLE
Why does 9.4.0 tag has data-import-handler-9.3.0.jar and not data-import-handler-9.4.0.jar

### DISCUSSION THREAD

**Discussion by hellomuru1** [rohitbemax/dataimporthandler#71] [General]
Why does 9.4.0 tag has data-import-handler-9.3.0.jar and not data-import-handler-9.4.0.jar? Also the 9.3 doesn't even open up https://github.com/SearchScale/dataimporthandler/tree/v9.3.0/repo ? Am i missing something?

**Comment by chatman** [comment]
Earlier, the repos directory used to have all the jar files as a way for
release. Now, the releases page works fine:

https://github.com/SearchScale/dataimporthandler/releases

**Reply by hellomuru1** [reply]
Thanks a lot, that was very timely!
